checker/src/main/java/org/checkerframework/checker/nonempty/NonEmptyTransfer.java

```diff
@@ -370,4 +393,20 @@ private Integer emptyValue(Node n) {
             return null;
         }
     }
+
+    /**
+     * If the transfer result given is regular result, then turn it into a conditional result while
+     * preserving the store. Otherwise, do nothing.
+     *
+     * @param result an input transfer result for regularity check
+     * @return a conditional transfer result
+     */
+    private TransferResult<CFValue, CFStore> regularCheck(TransferResult<CFValue, CFStore> result) {
+        if (!result.containsTwoStores()) {
+            CFStore thenStore = result.getThenStore();
+            CFStore elseStore = result.getElseStore();
```

Comment:
If `result` doesn't have two stores, what do `getThenStore` and `getElseStore` return? The same store or two different copies? Maybe add a comment to make this easier to read. Or copy one explicitly.